rust/matches/src/lib.rs

```diff
@@ -768,6 +911,47 @@ pub struct CreateOrUpdateOracle<'info> {
     rent: Sysvar<'info, Rent>,
 }
 
+#[derive(Accounts)]
+pub struct MatchJoinNamespace<'info> {
+    #[account(mut)]
+    match_instance: Account<'info, Match>,
+    #[account()]
+    namespace: UncheckedAccount<'info>,
+    #[account(address = anchor_lang::solana_program::sysvar::instructions::ID)]
+    instructions: UncheckedAccount<'info>,
+}
+
+#[derive(Accounts)]
+pub struct MatchLeaveNamespace<'info> {
+    #[account(mut)]
+    match_instance: Account<'info, Match>,
```

Comment:
https://github.com/raindrops-protocol/raindrops/pull/28/files#r945333657

Reply:
you cant use match, thats a reserved word in rust